pages/header.py

```diff
@@ -0,0 +1,20 @@
+from methods import Methods
+from locators import Locators
+
+class Header():
+    """This is parent class"""
+    def __init__(self, driver):
+        self.driver = driver
+        self.methods = Methods(self.driver)
```

Comment:
just an idea: make an inheritance for this class from Methods class and just use super().__init__(driver) and use methods of this class without self.methods

Reply:
Done. Commit amend is coming soon